lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -336,13 +489,10 @@ class MockTimers {
   }
 
   runAll() {
-    if (!this.#isEnabled) {
-      throw new ERR_INVALID_STATE(
-        'You should enable MockTimers first by calling the .enable function',
-      );
-    }
-
-    this.tick(Infinity);
+    this.#assertTimersAreEnabled();
+    const longestTimer = this.#executionQueue.peekBottom();
+    if (!longestTimer) return;
+    this.tick(longestTimer.runAt - this.#now);
```

Comment:
Sorry for being unclear - setTime shouldn't tick but ticking should progress time.

Reply:
Other than that mostly LGTM

Reply:
Oh ok! Sorry I thought it was all about ticking the time in general 😅

Reply:
Hey you resolved it but I still see setTime calling tick?

Reply:
I forgot to push the last changes, I was pushing all together but I didn't include it and went out, it should be there now!